integration-cli/docker_cli_run_test.go

```diff
@@ -3186,3 +3186,15 @@ func (s *DockerSuite) TestRunUnshareProc(c *check.C) {
 		c.Fatalf("unshare should have failed with permission denied, got: %s, %v", out, err)
 	}
 }
+
+func (s *DockerSuite) TestRunPublishPort(c *check.C) {
+	testRequires(c, NativeExecDriver)
```

Comment:
no, it's not a required, I use it on local test and forget to remove it when push to Github
thanks, I'll update